bench/wtperf/wtperf.h

```diff
@@ -223,6 +256,10 @@ struct __config_thread {		/* Per-thread structure */
 	TRACK insert;			/* Insert operations */
 	TRACK read;			/* Read operations */
 	TRACK update;			/* Update operations */
+	TRACK truncate;			/* Truncate operations */
+	TRACK truncate_sleep;		/* Truncate sleep operations */
+
+	uint64_t total_inserts;		/* Inserts done by this thread */
```

Comment:
Why is this different to the ```insert``` field? I believe that the ```TRACK insert``` field already has the behavior you want - i.e: A call to ```sum_insert_ops``` could replace tracking ```total_inserts```